docs/en/sql-reference/statements/detach.md

```diff
@@ -5,12 +5,66 @@ toc_title: DETACH
 
 # DETACH Statement {#detach}
 
-Deletes information about the ‘name’ table from the server. The server stops knowing about the table’s existence.
+Deletes information about the table or materialized view from the server. The server stops knowing about their existence.
```

Comment:
It doesn't delete anything. May be smth like "Unregister the table / Makes the server forget about the table"